packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsList.tsx

```diff
@@ -37,7 +39,13 @@ export const AccountsList = ({ onItemClick }: AccountListProps) => {
     const list = sortByCoin(accounts.filter(a => a.deviceState === device.state).concat(failed));
     const filteredAccounts =
         searchString || coinFilter
-            ? list.filter(a => accountSearchFn(a, searchString, coinFilter, accountLabels[a.key]))
+            ? list.filter(account => {
+                  const { key, accountType, symbol, index } = account;
+                  const accountLabel =
+                      accountLabels[key] || getDefaultAccountLabel({ accountType, symbol, index });
+
+                  return accountSearchFn(account, searchString, coinFilter, accountLabel);
```

Comment:
:thought_balloon:  I'd add a test case for this at [accountUtils.test.ts#L217](https://github.com/trezor/trezor-suite/blob/bd936caf71aaee85cbe4f6579a30069381401021/suite-common/wallet-utils/src/__tests__/accountUtils.test.ts#L217).
Although the `accountSearchFn` function is itself unchanged, its usage has slightly changed. There is now no test case to match substring with _supplied_ label, only whole string.
This will test it, and at the same time it'll help describe what is the fn typically used for :slightly_smiling_face: 
```
expect(accountSearchFn(btcAcc, '#1', undefined, 'Bitcoin #1')).toBe(true);
```